cmd/generate/generate.go

```diff
@@ -210,7 +210,12 @@ func Run(o *GenerateOptions) (err error) {
 		return fmt.Errorf("unsupported platform type: %s. Supported platform types are: %v", platformType, types.SupportedPlatforms)
 	}
 
-	matrix, err := generateMatrix(o, deployment, platformType)
+	ipv6Enabled, err := o.utilsHelpers.IsIPv6Enabled()
+	if err != nil {
+		log.Warnf("failed to detect IPv6: %v", err)
```

Comment:
why we warn here and not fail? also (sorry for not mentioning before), why is this not a flag but discovered by the code?